app/views/layout.erb

```diff
@@ -59,6 +59,9 @@
                           <a href="#friends" class="smoothScroll">Our Friends</a>
                         </li>
                         <li class="leaf">
+                          <a href="#rsvp" class="smoothScroll">RSVP</a>
```

Comment:
Do you want rspv on the same page or on a different page with just the rsvp form?
If the latter then change to /rsvp